### PR TITLE
Connect screen

### PR DESCRIPTION
**Pull request by vinnyjth**

### CODE REVIEW

**Comment by vinnyjth** [comment]
@conrad-vanl I can walk you through the contentful side of things at some point, if you wish. It should be pretty straightforward though.

**Comment by conrad-vanl** [comment]
Putting this on hold while we finish up design for connect

**Comment by jordanwade** [comment]
@vinnyjth Made some tweaks and added social icons. I also add a temporary array for the socialIcons for testing. The font will be fixed in the `add-fonts` PR/branch.
image

**Comment by vinnyjth** [comment]
🙌 🙌 🙌 🙌 🙌 🙌  I can't thank you enough.

**Comment by vinnyjth** [comment]
@conrad-vanl anything else you want in this PR before I merge it? 

It's got the social icons (driven by static data on the API), that action table is driven by a Contentful Page and it's attached data. Using a model identical to Conference.

**Comment by vinnyjth** [comment]
There's some weirdness because we are driving data from two different `contentful` instances. I'm going to try and unify them into a single `Class` where the "id" stores what instance the data came from. I had to break Conference `Links` to use the porch `Links` which obviously won't work well long term.

File: ThePorch/src/tabs/connect/ConnectScreenFeatures.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  justifyContent: 'center',
+  alignItems: 'center',
+}))(View);
+
+const StyledH2 = styled(({ theme }) => ({
+  fontSize: theme.sizing.baseUnit * 1.75,
+}))(H2);
+
+const SocialIconsFeature = ({ title, socialIcons }) => {
+  // TODO: Once `socialIcons` is working use that instead of `socialIconsTemp`. Doing this for testing purposes.
```
Comment (conrad-vanl) [inline ThePorch/src/tabs/connect/ConnectScreenFeatures.js:35]: ??

Comment (jordanwade) [reply ThePorch/src/tabs/connect/ConnectScreenFeatures.js:35]: When I was fixing the styles for the social icons the icon was hard coded to the `play` icon and it didn't seem like the `socialIcons` from the api was wired up yet. I'll double check with Vincent in the morning to see if that is wired up now. Then I can kill my hard coded array below that comment.

Comment (jordanwade) [reply ThePorch/src/tabs/connect/ConnectScreenFeatures.js]: @conrad-vanl I pushed up fix to use the static `socialIcons` array from the api Vincent setup. TODO comment is gone. https://github.com/watermarkchurch/theporch-apollos/pull/30/commits/dba800238d7a2c02f14d069890512100723b1719

**Comment by conrad-vanl** [comment]
@jordanwade does this still "Need changes"?

**Comment by jordanwade** [comment]
@conrad-vanl from a UI stand point I believe so 👍

**Comment by conrad-vanl** [comment]
@vinnyjth is this g2g?

**Comment by conrad-vanl** [comment]
@vinnyjth merge if so.

**Pull request merged**